### ISSUE TITLE
Fix test_should_pass_pbtest_org_suite in Firefox

### ISSUE THREAD

**Issue by ghostwords** [EFForg/privacybadger#1247]
Consistently failing:
```
selenium/pbtest_org_test.py::PBTest_Org_test::test_should_pass_pbtest_org_suite <- selenium/pbtest.py 
...
pbtest_org test results: 7 passed, 3 failed, 0 undefined
```

**Comment by cowlicks** [comment]
Related to #1185

**Comment by ghostwords** [comment]
OK, these are all passing now, after applying the fix in #1259.
